Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM rust:alpine AS builder
+
+RUN apk add python3 musl-dev pkgconfig openssl-dev make
+
+ENV RUSTFLAGS="-C target-feature=-crt-static"
+
+WORKDIR /opt/purginator3000/
```

Comment:
Is this directory name arbitrary? Could we change it to something more appropriate?

Reply:
@squahtx I used /opt/src/ in my local Dockerfile when testing [here ](https://github.com/matrix-org/rust-synapse-compress-state/pull/83#issuecomment-1038608156) and it worked fine. 😄